applications/holoviz/holoviz_srgb/holoviz_srgb.cpp

```diff
@@ -146,8 +148,8 @@ int main(int argc, char** argv) {
     const std::string argument(optarg ? optarg : "");
     switch (c) {
       case 'h':
-        std::cout << "Holoscan ClaraViz volume renderer."
-                  << "Usage: " << argv[0] << " [options]" << std::endl
+        std::cout << "Holoscan ClaraViz volume renderer." << "Usage: " << argv[0] << " [options]"
```

Comment:
I just noticed that this `Holoscan ClaraViz volume renderer.` is wrong. I replaced it with the actual example name and added a newline instead of a space.